Dockerfile

```diff
@@ -1,5 +1,15 @@
-# govukpay/nodejs:alpine-3.8.1
-FROM govukpay/nodejs@sha256:a3ae1b0b3ab5e7cdefe0787b97b1047cdc5e0c5166fb677e3cefac538bc99f86
+# alpine:3.8
+FROM alpine@sha256:ea47a59a33f41270c02c8c7764e581787cf5b734ab10d27e876e62369a864459
```

Comment:
Is there an advantage to using alpine + add `npm`, `nodejs` over the nodejs alpine images? 
https://github.com/nodejs/docker-node/blob/5a6a5e91999358c5b04fddd6c22a9a4eb0bf3fbf/10/alpine/Dockerfile

Reply:
I don't know - this is directly equivalent to what we do today, just importing the code from the current base image `Dockerfile` in `pay-infra`